.github/workflows/super-linter.yml

```
@@ -0,0 +1,40 @@
+name: Static analysis and fixers
+
+on:
```

yamllint

[warning] truthy value should be one of [false, true]

(truthy)